### PR TITLE
增加helm 包管理器

### PR DESCRIPTION
**Pull request by arrowfeng**

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (fdiskbrain) [inline helm/Chart.yaml]:
```suggestion
description: A Helm chart for nacos
```

File: helm/charts/mysql/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (fdiskbrain) [inline helm/charts/mysql/Chart.yaml]:
```suggestion
description: A Helm chart for nacos-mysql
```

**Comment by g5niusx** [comment]
👍

**Comment by paderlol** [comment]
Many thanks for your contribution @arrowfeng

**Pull request merged**